xml/System.Windows.Forms/MdiClient.xml

```diff
@@ -325,9 +318,9 @@
         <Parameter Name="dy" Type="System.Single" />
       </Parameters>
       <Docs>
-        <param name="dx">To be added.</param>
-        <param name="dy">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="dx">The ratio by which to scale the control horizontally.</param>
+        <param name="dy">The ratio by which to scale the control vertically.</param>
+        <summary>Scales the entire control and any child controls.</summary>
```

Comment:
@RussKie @Tanya-Solyanik can you confirm whether this correct? This comment makes me think that it could be wrong

https://source.dot.net/#System.Windows.Forms/System/Windows/Forms/MDIClient.cs,185

Reply:
Interesting! SetBounds scales the child controls as a side affect, so I'm not sure what the comment on line 185 means...

Do we say "the *ratio* by which to scale" or "the *factor* by which to ..."

Reply:
I would not say vertically/horizontally, but ` along the x-axis` and `along the y-axis`

Reply:
I'll provide your suggestions, @Tanya-Solyanik , on the appropriate lines above.

Reply:
I talked to @rpetrusha and we decided to standardize on factor. I'll open a new PR to fix existing descriptions for other APIs later today.